src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -162,7 +163,7 @@ public void update(StandaloneConfig standaloneConfig) {
     }
 
     public String getRepoRoot() {
-        String path = FileUtil.REPOS_ADDRESS + File.separator + displayName + File.separator;
+        String path = FileUtil.REPOS_ADDRESS + File.separator + repoName + File.separator;
 
         if (!repoName.isEmpty()) {
             path += repoName + File.separator;
```

Comment:
This portion should be redundant after change from `displayName` to `repoName` in Line 166

It should be safe to removed as `repoName` is initialized in the constructor, thus it cannot be empty.

Reply:
@yong24 removing the check causes many of the tests to fail, it would be good to create this as a separate issue instead

Reply:
i think this is related to #378